src/main/java/org/fundacionjala/coding/kevinherrera/crypto/INormalCypher.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.coding.kevinherrera.crypto;
+
+/**
+ * Interface for normal cypher.
+ */
+public interface INormalCypher {

```

Comment:
this class is redundant.
Please move the abstract methods (encode, decode) inside CesarCrypto class